azure/pipelines/dotnet/ci.yml

```diff
@@ -62,6 +62,10 @@ parameters:
   type: boolean
   default: true
   displayName: 'Whether to cache NuGet packages. NUGET_PACKAGES environment variable still needs to be set though.'
+- name: InternalPublishFeedName
```

Comment:
I suggest to rename this to InternalFeedName and move it just before ExternalFeedCredentials